### PR TITLE
Static fix

### LINKED ISSUE #87: Image file contents not passed into public
After compilation, the image contents don't pass through to the files that appear in the `public` folder.

### PR DESCRIPTION
**Pull request by jenius**
Quick fix, and a test case to cover it for the future!

Closes #87

### CODE REVIEW

File: lib/plugins/static_plugin.js [hunk 1]
```diff
@@ -42,7 +42,7 @@ export default class StaticWebpackPlugin {
             .replace(this.opts.root, '')
 
           // now we pull the value from thr dependency using the correct key
-          src = String(dep.assets[relativePath]._value)
+          src = dep.assets[relativePath]._value
```
Comment (kylemac) [inline lib/plugins/static_plugin.js]: ha yeah, that makes sense.

**Comment by kylemac** [comment]
LGTM

**Comment by hhsnopek** [comment]
Looks like there's a spelling error in the quoted text 'thr dependencies' -
at least it shows in the email

-- 
Henry Snopek

**Comment by hhsnopek** [comment]
There's no back tick on my iPhone, but here's the section that I think has
the spelling error, sorry if I'm wrong!
"""
> @@ -42,7 +42,7 @@ export default class StaticWebpackPlugin {

>              .replace(this.opts.root, '')
>
>            // now we pull the value from thr dependency using the correct key

"""


-- 
Henry Snopek

**Comment by kylemac** [comment]
merge n release; and lets get the source-loader out there too

**Pull request merged**